g2p/cli.py

```diff
@@ -35,20 +35,31 @@ def cli():
     '''Management script for G2P'''
 
 
-@click.option('--out-dir', type=click.Path(exists=True, file_okay=False, dir_okay=True))
-@click.option('--ipa/--no-ipa', default=False)
-@click.option('--dummy/--no-dummy', default=False)
+@click.option('--out-dir', type=click.Path(exists=True, file_okay=False, dir_okay=True),
+    help='Output results in DIRECTORY instead of the global "generated" directory.')
+@click.option('--list-dummy', default=False, is_flag=True, help="List the dummy phone inventory.")
+@click.option('--dummy/--no-dummy', default=False, help="Generate dummy fallback mapping to minimalist phone inventory.")
+@click.option('--ipa/--no-ipa', default=False, help="Generate mapping from LANG-ipa to eng-ipa.")
 @click.argument('in_lang', type=click.Choice([x for x in LANGS_NETWORK.nodes if not is_ipa(x) and not is_xsampa(x)]))
-@cli.command(context_settings=CONTEXT_SETTINGS)
-def generate_mapping(in_lang, dummy, ipa, out_dir):
-    ''' Generate English mapping.
+@cli.command(context_settings=CONTEXT_SETTINGS, short_help="Generate English mapping.")
```

Comment:
Add a description, maybe with better wording that this:

```suggestion
@cli.command(context_settings=CONTEXT_SETTINGS, short_help="Generate English IPA or dummy mapping.")
```